packages/vscode-extension/src/handlers.ts

```diff
@@ -226,6 +226,7 @@ export async function migrateV1ProjectHandler(args?: any[]): Promise<Result<null
   );
   const result = await runCommand(Stage.migrateV1);
   await openMarkdownHandler();
+  await vscode.commands.executeCommand("setContext", "fx-extension.sidebarWelcome", false);
```

Comment:
Even you set"fx-extension.sidebarWelcome" context to false, the tree view still will not be updated. If the user is not in sidebar welcome treatment group, he still can't see tree items like 'provision', 'deploy',...

Reply:
Because the migrate from v1 project cannot provision, deploy, ......, may be we can keep this logic and discuss a better way to fix it in the next sprint.